src/modm/platform/clock/samg/module.lb

```diff
@@ -15,7 +15,8 @@ def init(module):
     module.description = "Clock Generator (CKGR)"
 
 def prepare(module, options):
-    if not options[":target"].has_driver("pmc:samg*"):
+    if not (options[":target"].has_driver("pmc:samg*")
```

Comment:
@mcbridejc  do we need a wildcard here?

Reply:
Currently yes. The samg55, samg54, etc are marked with specific types. I don't know how much peripheral overlap there is between them, so went granular so drivers could distinguish in the future if necessary.